match2/wikis/rocketleague/legacy_match_list.lua

```diff
@@ -233,6 +239,12 @@ function LegacyMatchList.getMapScoreFromGoals(goals)
 	return #indexedGoals
 end
 
+function LegacyMatchList.dateStuff(args, details)
```

Comment:
Can we make this more specific?

Reply:
i couldn't think of a nice name ...
it sets date into header of header is empty
and it also sets args.date from details if present there or else from args

Reply:
```suggestion
function LegacyMatchList.setHeaderIfEmpty(args, details)
```